.github/workflows/create-and-publish-docker-manually.yml

```diff
@@ -6,65 +6,23 @@ on:
       version:
```

Comment:
Maybe call this workflow `create-and-publish-docker-dev-manually` or so? Just to make it clear that this is not a manual release but a manual release of the dev image (if I read it correctly, that is 😄 ).

Reply:
Yes, that is what it is doing currently. I want to adapt and possibly remove this when it is not needed any longer. Will rename it!